src/components/app/CompareHome.css

```diff
@@ -11,6 +11,7 @@
   background: #fff;
   border-radius: 3px;
   box-shadow: 0 5px 25px #0b1f50;
+  font-size: 130%;
```

Comment:
This is to align the header size with the home page. Ideally we should remove this and use the new typography classes instead, but this is more work than I wanted to do.